.github/workflows/pr-style.yml

```diff
@@ -19,4 +19,4 @@ jobs:
         with:
           require_issue: "true"
           skip_authors: "dependabot"
-          title_prefixes: " [repo] bump, [repo] Bump, [repo] chore:, [repo] docs:, [repo] fix:, [JS] bump, [JS] Bump, [JS] chore:, [JS] docs:, [JS] feat:, [JS] fix:, [JS] port:, [JS] refactor:, [C#] bump, [C#] Bump: [C#] chore:, [C#] docs:, [C#] feat:, [C#] fix:, [C#] port:, [C#] refactor:, [PY] bump, [PY] Bump, [PY] chore:, [PY] docs:, [PY] feat:, [PY] fix:, [PY] port:, [PY] refactor:"
+          title_prefixes: " [repo] bump, [repo] Bump, [repo] chore:, [repo] docs:, [repo] fix:, [JS] bump, [JS] Bump, [JS] chore:, [JS] docs:, [JS] feat:, [JS] fix:, [JS] port:, [JS] refactor:, [C#] bump:, [C#] Bump:, [C#] chore:, [C#] docs:, [C#] feat:, [C#] fix:, [C#] port:, [C#] refactor:, [PY] bump, [PY] Bump, [PY] chore:, [PY] docs:, [PY] feat:, [PY] fix:, [PY] port:, [PY] refactor:"
```

Comment:
JW, I know I did all these strings but is there a way to do it more regex-y in yml?

Reply:
@corinagum thats up to the action, so if the pr-style action input `title_prefixes` supports a regex input instead of string then we could do that.

Reply:
Will look into this later :) thanks